content/help/ops/traffic-management/troubleshooting/index.md

```diff
@@ -9,3 +9,36 @@ weight: 5
 * Provide a few general procedures that should be followed to isolate
 
 * Describe high level isolation steps and things to check.
+
+## Pod reachability issues
+
+Usually, the first thing you want to check is whether or not the POD associated with your app has been allocated with a valid IP address. This can be done by examining the POD status with `kubectl get pods -o wide`. In addition, check whether or not the service associated with the app has been properly created with a service IP. Assuming the POD is up and running and IP is allocated, ping the POD IP and make sure it can be pinged. Otherwise, a few things that may be done to isolate the issue:
```

Comment:
allocated with a valid IP address/allocated a valid IP address.

Reply:
and an IP is allocated

Reply:
Remove last sentence.

Add a new paragraph:
If neither of these trouble shooting steps indicate a problem, try: